site.mk

```diff
@@ -15,19 +14,11 @@ GLUON_FEATURES := \
 	web-private-wifi \
 	web-logging \
 
-
 GLUON_SITE_PACKAGES := \
 	iwinfo \
-	iptables \
```

Comment:
Does not affect size, however selected as a dependency of `gluon-mesh-vpn-core` already.

Reply:
So can still be drooped out of here... Even when it does not save space